tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -295,7 +295,7 @@ def fit(train_data,
         stack_ensemble_levels : int, default = 0
             Number of stacking levels to use in stack ensemble. Roughly increases model training time by factor of `stack_ensemble_levels+1` (set = 0 to disable stack ensembling).
             Disabled by default, but we recommend values between 1-3 to maximize predictive performance.
-            To prevent overfitting, this argument is ignored unless you have also set `num_bagging_folds >= 2`.
+            To prevent overfitting, this argument is ignored unless you have also set `num_bagging_folds >= 2`.
```

Comment:
What is this out of curiousity? I also see this edit when I delete this space and add it back.

Reply:
Was the weirdest thing. I think pasting it into vim made some sort of slight formatting change? I couldn't "fix it", but the space is there when I look at the code so it seems fine?